packages/orbit-components/src/Radio/index.tsx

```diff
@@ -64,18 +64,21 @@ const Radio = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
             "flex flex-none items-center justify-center",
             "size-icon-medium rounded-full",
             "duration-fast scale-100 transition-all ease-in-out",
-            "lm:border border-[2px] border-solid",
+            "lm:border border-solid",
+            checked ? "border-[2px]" : "border-[1px]",
```

Comment:
On large mobile or greater and checked, `lm:border` takes preference over `border-[2px]` (mq specificity I assume), resulting in a 1px border when it should be 2px.

<img width="420" alt="image" src="https://github.com/kiwicom/orbit/assets/8863238/41c345c2-bdc1-475f-875c-e3ee47f2875f">

Reply:
Well spotted. There does not seem to be a responsive difference now, so I have adjusted the classes. I have also used the native `border` and `border-2` tailwind classes, avoiding custom ones 🙂